VENCore.podspec

```diff
@@ -1,19 +1,14 @@
 Pod::Spec.new do |s|
-  s.name         = "VENCore"
-  s.version      = "3.1.1"
-  s.summary      = "Core Venmo client library"
-  s.description  = <<-DESC
-                   Core iOS client library for the Venmo api
-                   DESC
-
-  s.homepage     = "https://github.com/venmo/VENCore"
+  s.name         = 'VENCore'
+  s.version      = '3.1.2'
+  s.summary      = 'Core Venmo client library'
+  s.description  = 'Core iOS client library for the Venmo api'
+  s.homepage     = 'https://github.com/venmo/VENCore'
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
-  s.author       = { "Venmo" => "ios@venmo.com" }
+  s.author       = { 'Venmo' => 'ios@venmo.com' }
   s.platform     = :ios, '6.0'
-  s.source       = { :git => "https://github.com/venmo/VENCore.git",
-                     :tag => "v#{s.version}"
-  }
+  s.source       = { :git => 'https://github.com/venmo/VENCore.git',
+                     :tag => "v#{s.version}" }
   s.source_files = 'VENCore/**/*.{h,m}'
-  s.dependency 'CMDQueryStringSerialization', '~>0.2.0'
-  s.requires_arc = true
+  s.dependency 'CMDQueryStringSerialization'
```

Comment:
@eliperkins so the reason for this is I want to use version 0.3.0 in the Venmo app. I spoke with @calebd and he ensured me that the public API for serialization wouldn't change. To use 0.3.0, I had to remove the version restriction in `VENCore`, then update one of our private pods that had a version restriction on `VENCore` and then update all three in our app's Podfile. If we know this API won't change, I think it's best to be able to update it independently.

That said, @dasmer, if we only use it in one spot, maybe it's worth adding that specific category so we don't have an dependencies on something that should be super portable.

What do you guys think?